lib/ansible/utils/unicode.py

```diff
@@ -215,7 +215,7 @@ def to_bytes(obj, encoding='utf-8', errors='replace', nonstring=None):
         return obj
     elif nonstring == 'simplerepr':
         try:
-            simple = binary_type(obj)
+            simple = str(obj)
         except UnicodeError:
```

Comment:
There's code below this bit (lines 229-230) that converts text to bytes, if that's the case:
```python
         if isinstance(simple, text_type):
             simple = simple.encode(encoding, 'replace')
```